source-oracle/backfill.go

```diff
@@ -183,6 +208,122 @@ func (db *oracleDatabase) WatermarksTable() sqlcapture.StreamID {
 	return sqlcapture.JoinStreamID(bits[0], bits[1])
 }
 
+func (db *oracleDatabase) fetchBackfillRowIDRange(ctx context.Context, schemaName string, tableName string) error {
```

Comment:
Instead of passing in the schema/table name strings separately, this could just be the stream ID. 
(I recently got around to refactoring things so that it's a `struct{Schema, Table}` so you can use `streamID.Schema` and `streamID.Table` as needed)

It's also fine if you want to leave it like this though, there's plenty of other places we're still threading the two values around separately.

Reply:
I tried this but ran into lowercasing, I tried to prevent that by setting `sqlcapture.LowercasingStreamID` to false, it broke tests so I just gave up for now, we can revisit later